src/main/kotlin/com/cosmotech/api/security/AbstractSecurityConfiguration.kt

```diff
@@ -107,6 +111,30 @@ const val PATH_WORKSPACES = "/organizations/*/workspaces"
 const val PATH_WORKSPACES_USERS = "/organizations/*/workspaces/*/users"
 val PATHS_WORKSPACES = listOf(PATH_WORKSPACES, PATH_WORKSPACES_USERS)
 const val PATH_WORKSPACES_FILES = "/organizations/*/workspaces/*/files"
+// Job
+const val PATH_JOB_STATUS = "/organizations/*/job/*/status"
+val PATHS_JOB = listOf(PATH_JOB_STATUS)
+// Twingraph
+const val PATH_TWIN_GRAPH_IMPORT = "/organizations/*/twingraph/import"
+const val PATH_TWIN_GRAPH = "/organizations/*/twingraph"
+const val PATH_TWIN_GRAPHS = "/organizations/*/twingraphs"
+const val PATH_TWIN_GRAPH_QUERY = "/organizations/*/twingraph/*/query"
+const val PATH_TWIN_GRAPH_BATCH_QUERY = "/organizations/*/twingraph/*/batch-query"
+const val PATH_TWIN_GRAPH_BULK_DOWNLOAD = "/organizations/*/twingraph/download"
+const val PATH_TWIN_GRAPH_ENTITY = "/organizations/*/twingraph/*/entity"
+const val PATH_TWIN_GRAPH_METADATA = "/organizations/*/twingraph/*/metadata"
+const val PATH_TWIN_GRAPH_BATCH_ACTIONS = "/organizations/*/twingraph/*/batch"
```

Comment:
`const val PATH_TWIN_GRAPH_BATCH_ACTIONS = "/organizations/*/twingraph/*/upload"`

Reply:
`PATH_TWIN_GRAPH_BATCH_ACTIONS` is `/organizations/*/twingraph/*/batch` in main branch. Seems also to fit the needs :sweat_smile: